script/script.go

```diff
@@ -677,6 +683,30 @@ func setRequestHeader(f filters.FilterContext) func(*lua.LState) int {
 	}
 }
 
+func addRequestHeader(f filters.FilterContext) func(*lua.LState) int {
+	return func(s *lua.LState) int {
+		value := s.ToString(-1)
```

Comment:
minor: where did -1 and -2 came from? Can I use -3 and -4, for example?

Reply:
Ah, okay, I see, there is stack-based API.
https://pkg.go.dev/github.com/yuin/gopher-lua@v1.1.1#section-readme